conf/layer.conf

```diff
@@ -10,7 +10,7 @@ BBFILE_PATTERN_swupdate-dev = "^${LAYERDIR}/"
 BBFILE_PRIORITY_swupdate-dev = "6"
 
 LAYERDEPENDS_swupdate-dev = "core"
-LAYERSERIES_COMPAT_swupdate-dev = "thud warrior"
+LAYERSERIES_COMPAT_swupdate-dev = "zeus thud warrior"
```

Comment:
should be `zeus` only. This layer is no longer compatible with warrior since
we are using a specific U-Boot version for the stm32mp1.